packages/subiquity_client/lib/subiquity_server.dart

```diff
@@ -94,7 +94,9 @@ abstract class SubiquityServer {
       stderr.addStream(process.stderr);
       return process;
     });
-    log.info('Starting server (PID: ${_serverProcess!.pid})');
+    log.info(
+      'Starting server (PID: ${_serverProcess!.pid}) with args: $subiquityCmd',
```

Comment:
this is nice. :+1: it would be nice to have it in a separate commit, though

Reply:
I'll do some git tricks to split that commit in two.